### PR TITLE
Remove unused imports from corehq/apps/app_manager/

### PR DESCRIPTION
**Pull request by nickpell**
@dannyroberts cc: @kaapstorm

### CODE REVIEW

**Comment by kaapstorm** [comment]
Looks real.
```
  File "/mnt/corehq/apps/app_manager/tests/test_views.py", line 9, in <module>
    from corehq.apps.app_manager.views import AppSummaryView
ImportError: cannot import name AppSummaryView
```

File: corehq/apps/app_manager/views/__init__.py [hunk 1]
```diff
@@ -21,7 +21,6 @@
     download_xform,
     DownloadCCZ,
 )
-from corehq.apps.app_manager.views.app_summary import AppSummaryView
```
Comment (dannyroberts) [inline corehq/apps/app_manager/views/__init__.py]: Looks like this is causing the build error

**Comment by dannyroberts** [comment]
👍 on ✅

**Comment by nickpell** [comment]
✅

**Pull request merged**